README.md

```diff
@@ -281,3 +281,17 @@ purposes, since Pebble and its generated keys are not audited or held to the
 same standards as the Let's Encrypt production CA and their keys. Moreover
 these keys are exposed by Pebble and will be lost as soon as the process
 terminates: so they are not safe to use for anything other than testing.**
+
+### OCSP Responder URL
+
+Pebble does not support the OCSP protocol as a responder and so does not set
+the OCSP Responder URL in the issued certificates. However, if you setup a
+proper OCSP Responder run side by side with Pebble, you may want to set this URL.
+This is possible by setting the environment variable `PEBBLE_CA_OCSP_RESPONDER_URL`:
```

Comment:
This feels like a configuration that would be at home in the `pebble-config.json` file instead of an env var. The other env vars are related to runtime behaviour changes and feel like things someone could want to toggle quickly at the command line. The OCSP responder URL is likely to be set once, per-project and not changed. In that respect I think it fits better in the config file.

@adferrand What do you think?

Reply:
Initially I implemented this as a flag to pass to the command line. Then I thought it was not worth a fully-fledged flag changing pebble's command API for something which is nonetheless a corner case. So I moved to a environment variable.

But you are right, it is too much ephemeral like this. Your proposition to pass it in the configuration file seems a good compromise. I will do that.

Reply:
Implemented.